lib/bluetooth_device.js

```diff
@@ -1,5 +1,8 @@
 /* global Bluetooth, EventEmitter2 */
 'use strict';
+if (process) {
```

Comment:
When using browserify, we'll need require statements to pack all related node.js packages to the web, this is the easist way I can think of now to do it.